discretize/MeshIO.py

```diff
@@ -244,13 +244,13 @@ def writeVTK(mesh, fileName, models=None, directory=''):
 
         # Assign the model('s) to the object
         if models is not None:
-            for item in six.iteritems(models):
+            for key in models:
                 # Convert numpy array
-                vtkDoubleArr = numpy_to_vtk(item[1], deep=1)
-                vtkDoubleArr.SetName(item[0])
+                vtkDoubleArr = numpy_to_vtk(models[key], deep=1)
+                vtkDoubleArr.SetName(key)
                 vtkObj.GetCellData().AddArray(vtkDoubleArr)
             # Set the active scalar
-            vtkObj.GetCellData().SetActiveScalars(list(models.keys())[0])
+            vtkObj.GetCellData().SetActiveScalars(key)
```

Comment:
This line is outside the `for` loop, previously it was setting the first model in the dictionary as the active model. Here it is now setting the active model as the last value that `key` had in the for loop, (which was the last model in the dictionary)

Reply:
It doesn't really effect anything though